xgcm/test/test_metrics.py

```diff
@@ -163,3 +163,16 @@ def test_set_metric():
 
         for metric_expected, metric in zip(v, grid._metrics[k]):
             xr.testing.assert_equal(metric_expected.reset_coords(drop=True), metric)
+
+
```

Comment:
We could parametrize this test, so that we can check this for more than one predefined metric. Happy to help with that.